db/primary_migrate/20230419190148_add_gpo_verification_pending_to_profile.rb

```diff
@@ -0,0 +1,8 @@
+class AddGpoVerificationPendingToProfile < ActiveRecord::Migration[7.0]
+  disable_ddl_transaction!
+
+  def change
+    add_column :profiles, :gpo_verification_pending, :boolean
```

Comment:
Do we want this to be a boolean or a timestamp?

Reply:
I'm about to update the migration to be a timestamp